src/View/html/GravityForms/fieldset.php

```diff
@@ -44,7 +44,8 @@
 			</legend>
 
 			<span class="gform-settings-panel__collapsible-control">
-				<input type="checkbox" class="gform-settings-panel__collapsible-toggle-checkbox" name="<?= esc_attr( $collapsible_name ) ?>" id="<?= esc_attr( $collapsible_name ) ?>" value="1" onclick="this.checked ? this.closest( '.gform-settings-panel' ).classList.add( 'gform-settings-panel--collapsed' ) : this.closest( '.gform-settings-panel' ).classList.remove( 'gform-settings-panel--collapsed' )" <?php checked( empty( $args['collapsible-open'] ) ); ?>>
+				<input type="checkbox" class="gform-settings-panel__collapsible-toggle-checkbox" name="<?= esc_attr( $collapsible_name ) ?>" id="<?= esc_attr( $collapsible_name ) ?>" value="1" onclick="this.checked ? this.closest( '.gform-settings-panel' ).classList.add( 'gform-settings-panel--collapsed' ) : this.closest( '.gform-settings-panel' ).classList.remove( 'gform-settings-panel--collapsed' )" onkeydown="if(event.keyCode==13){ event.preventDefault(); this.click(); }"
+					<?php checked( empty( $args['collapsible-open'] ) ); ?>>
```

Comment:
Best to have each attribute begin on a new line for easier scanning.

Reply:
Added new line on both enter and click events.